### PR TITLE
Erd creation

### LINKED ISSUE #16: Creating ERD for MVP
Implementing the ERD needed for the MVP, with the tables : 
-Item
-Business
-Consumer
-Happy Price
-Happy Code
-Administrator 
(Owner is already coded)

### PR DESCRIPTION
**Pull request by Vivien-Ploix**
Closes #16 

Creation of ERD models : 

-Administrator
-Consumer
-Business
-HappyPrice
-HappyCode
-Item

### CODE REVIEW

File: app/models/consumer.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Consumer < ApplicationRecord
```
Comment (mathieu-superpose) [inline app/models/consumer.rb:1]: Shouldn't we call it Customer?

Comment (Vivien-Ploix) [reply app/models/consumer.rb:1]: That's what I had in mind but it appears the others chose to call it this way in the ERD diagram, we'll see what their idea on the subject is tomorrow morning

Comment (mvoland) [reply app/models/consumer.rb:1]: I think we decided Customer? I think either one is OK we just need to use the same everywhere

Comment (Laurent-Gi) [reply app/models/consumer.rb:1]: Yes, I remember of Customer, but no problem to change if we all agreed

File: app/models/happy_price.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class HappyPrice < ApplicationRecord
+  belongs_to :item
+  has_many :happy_prices
```
Comment (Laurent-Gi) [inline app/models/happy_price.rb:3]:
```suggestion
  has_many :happy_codes
```

**Review by Laurent-Gi — CHANGES_REQUESTED** [review]

**Comment by mvoland** [comment]
[Modèles et migrations pour Business, ITem, Customer, Happy Price, Happy Code, Administrator](https://trello.com/c/WsS7HKeL/54-mod%C3%A8les-et-migrations-pour-business-item-customer-happy-price-happy-code-administrator)

**Review by StanislasBASQUIN — APPROVED** [review]
Review Ok

**Pull request merged**